code/modules/research/designs/boards/machine_engie.dm

```diff
@@ -46,14 +46,24 @@
 
 /datum/design/cell_charger
 	name = "Circuit Design (Cell Charger)"
-	desc = "Allows for the construction of circuit boards used to build a cell charger"
+	desc = "Allows for the construction of circuit boards used to build a Cell Charger."
```

Comment:
why

Reply:
Unatomic but it's just to bring that bit of text more in line with other instances of that style.